commands/xautoclaim.md

```diff
@@ -19,7 +19,7 @@ However, note that you may want to continue calling `XAUTOCLAIM` even after the
 Note that only messages that are idle longer than `<min-idle-time>` are claimed, and claiming a message resets its idle time.
 This ensures that only a single consumer can successfully claim a given pending message at a specific instant of time and trivially reduces the probability of processing the same message multiple times.
 
-While iterating the PEL, if `XAUTOCLAIM` stumbles upon a message which doesn't exist in the stream anymore (either trimmed or deleted by `XDEL`) it does not claim it, and deletes it from the PEL in which it was found. This feature was introduced in Redis 7.0.
+While iterating PEL, if `XAUTOCLAIM` stumbles upon a message which doesn't exist in the stream anymore (either trimmed or deleted by `XDEL`) it does not claim it, and deletes it from the PEL in which it was found. This feature was introduced in Redis 7.0.
```

Comment:
```suggestion
While iterating the PEL, if `XAUTOCLAIM` stumbles upon a message which doesn't exist in the stream anymore (either trimmed or deleted by `XDEL`) it does not claim it, and deletes it from the PEL in which it was found. This feature was introduced in Redis 7.0.
```

Reply:
it iterates the PEL of the whole group (which is comprised of individual consumer PELs)